packages/dd-trace/test/plugin_manager.spec.js

```diff
@@ -338,4 +338,70 @@ describe('Plugin Manager', () => {
       expect(Four.prototype.configure).to.have.been.calledWithMatch({ enabled: false })
     })
   })
+
+  describe('_getSharedConfig', () => {
```

Comment:
Can a test be added to test the behaviour instead? The fact that we test this with mocked unit tests is why this wasn't caught in the first place.

Reply:
I meant testing with an actual web server whether the config applies, for example in one of our plugin tests.

Reply:
Done 🏁

Reply:
I would also remove this test at that point as I don't think it bring any more value that the newly added `http` test.

Reply:
done, should the web test be removed as well? And just keep the koa plugin test?

Reply:
Nah that one is fine, it's an actual util used elsewhere and not just an internal function.